package.json

```diff
@@ -26,6 +31,19 @@
 		"dotenv": "^6.1.0",
 		"express": "^4.16.4",
 		"morgan": "^1.9.1",
-		"parse-error": "^0.2.0"
+		"parse-error": "^0.2.0",
+		"react": "^16.5.2",
+		"react-dom": "^16.5.2",
+		"rimraf": "^2.6.2",
+		"webpack": "^3.12.0"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.1.2",
+		"@babel/preset-env": "^7.1.0",
+		"@babel/preset-react": "^7.0.0",
+		"babel-cli": "^6.26.0",
```

Comment:
I think this should be @babel/cli not babel-cli because of the version 7 migration. I was getting errors when I would try to do npm run build-babel-server, but when I changed it the problems were gone. It also upgrades it to 7.1.2 and the problems I was getting were related to the version not being 7.x. [This article](https://babeljs.io/docs/en/v7-migration) explains this under the Scoped Packages heading.

Reply:
Nice ok, I can upgrade that package